libs/payments/cart/src/lib/cart.error.ts

```diff
@@ -200,6 +200,16 @@ export class CartNoTaxAddressError extends CartError {
   }
 }
 
+export class CartIntentNotFoundError extends CartError {
+  constructor(cartId: string) {
+    super('Cart payment or setup intent not found', {
+      cartId,
+    });
+    this.name = 'CartIntentNotFoundError';
+    Object.setPrototypeOf(this, CartSubscriptionNotFoundError.prototype);
```

Comment:
This pattern was due to a misunderstanding I had of the problems with `instanceof` checking across the server-client boundary. I'm removing this statement on the other errors in my current PR, but we can remove it from this as well